src/core/objects/PDFDict.ts

```diff
@@ -45,6 +46,8 @@ class PDFDict extends PDFObject {
     return this.dict.has(key);
   }
 
+  lookupMaybe(key: PDFName): PDFObject | undefined;
```

Comment:
I think this method is redundant, given that looking up a key with no type specified already returns an optional value: 

https://github.com/Hopding/pdf-lib/blob/620747a044c2b3864c9d844b7c67b924b4f94ea5/src/core/objects/PDFDict.ts#L66

Reply:
Updated